.idea/encodings.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
IntelliJ configuration cannot be accepted into a pull request. Please remove them from your pull request.